cylc/flow/etc/syntax/cylc.lang

```diff
@@ -136,7 +136,6 @@
         <keyword>number of cycle points</keyword>
         <keyword>members</keyword>
         <keyword>max-polls</keyword>
-        <keyword>max active cycle points</keyword>
```

Comment:
Should these syntax highlighting keywords be removed or not until the setting is obsoleted?

Reply:
Can't remember if we've done this consistently in the past.  But IMO yes, it might provide additional incentive to upgrade deprecated items.

Reply:
(we will need to re-do these at some point)

Reply:
I'll open an issue for that?

Reply:
(I see you've done that #3860 )